server/routes/contact.js

```diff
@@ -24,9 +24,15 @@ module.exports = (userAdminService, roService) => router => {
         return res.render('contact/ro', { contact })
       }
 
+      const functionalMailbox = ro && ro.lduCode && (await userAdminService.getFunctionalMailbox(ro.lduCode))
```

Comment:
This is very similar to what happens in `roContactDetailsService.getResponsibleOfficerWithContactDetails()`

might be worth seeing if that has all the data you need here?